IPython/core/excolors.py

```diff
@@ -155,6 +156,12 @@ def exception_colors():
         Normal = C.Normal,
         ))
 
+    # Hack: the 'neutral' colours are not very visible on a dark background on
+    # Windows. Since Windows command prompts have a dark background by default, and
+    # relatively few users are likely to alter that, we will use the 'Linux' colours,
+    # designed for a dark background, as the default on Windows.
+    if os.name == "nt":
+        ex_colors.add_scheme(ex_colors['Linux'].copy('Neutral'))
```

Comment:
The problem is that the Linux color scheme uses monokai syntax highlighting which is weird. 
([IPython/terminal/interactiveshell.py:266-268](https://github.com/ipython/ipython/blob/b8cfbfe27bdac730111d32a29f8aab469c5fcf45/IPython/terminal/interactiveshell.py#L266-L268)).

I think what we really want is the Linux color scheme with the Pygments default syntax highlighting with the style overrides used in Neutral. (With fixes for Windows when running on Windows)